src/aqua_diagnostics/ocean_trends/plot_trends.py

```diff
@@ -149,6 +162,8 @@ def set_ytext(self):
 
     def set_data_list(self):
         """Prepare the list of data arrays to plot."""
+        self.logger.debug('Loading the data into memory')
+        self.data = self.data.compute()
```

Comment:
Please remove these two lines, we deal with loading the data in #2371. Let's wait to have all together and then we can make further adjustments.